src/main/java/com/hpe/adm/octane/ideplugins/services/connection/IdePluginsOctaneHttpClient.java

```diff
@@ -173,15 +174,25 @@ private boolean octaneAuthenticate(Authentication authentication) {
 			throw new RuntimeException(e);
 		}
 	}
+
+	public boolean isAuthenticated() {
```

Comment:
is this method used anywhere else? Not really needed in this case

Reply:
planning to use it later